README.md

```diff
@@ -149,8 +153,87 @@ See [attribute referencing](https://amzn.github.io/style-dictionary/#/properties
         'on-dark': {
           value: '{options.color.heart-of-darkness}',
           category: 'color',
-          docCategory: 'colors',
-          docExample: 'color'
+          docs: {
+            category: 'colors',
+            type: 'text',
+            example: 'color'
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Documentation within tokens
+
+Tokens also have data that can be added to them to help generate documentation & examples. To get an idea of how this data is used see [the tokens example page](https://rei.github.io/rei-cedar-tokens/). This data is mostly used to create groupings of tokens and is not used to do transforms on token values, only for display in docs.
+
+The docs object looks like this:
+
+```js
+docs: {
+  category: String,     // Large, broad groupings of things (i.e. color, spacing)
+  type: String,         // Sub category (i.e. background (color) or inset (spacing))
+  example: String,      // Used to determine how to display a visual representation of a token
+                        // Current options (see docs/src/components/PropSorter.vue): color, spacing, sizing, radius, prominence, text, inset, breakpoint, timing, duration, an empty example defaults to 'token' which is just a string representation of the value.
+  description: String,  // Short description of the token and/or suggested usage (displayed on the cedar docs site)
+}
+```
+
+### Deprecating tokens
+
+Deprecated tokens should be moved to a seprate file (or into the existing file) which corresponds to the release cycle in which they will be deprecated.
+
+For example, if tokens will be considered deprecated in the "Winter 2019" release they would be moved into a file called `deprecated-2019-winter.json5` in whichever directory they currently reside. Structure for naming the file is : `deprecated-<year>-<release>`
+
+Additionally, the contents will be wrapped inside an object with a key that corresponds to the release as well (so we can auto generate some deprecation warnings with the correct release). The key matches the naming of the file. See below for an example.
+
```

Comment:
Do we have any guidelines on what to do to the value of the deprecated token?  Do we map it to new value? Keep it as the original value? Is it a case by case decision?  Mostly wondering if there is a convention to follow.

Reply:
I believe the recommendation is to map it to a new value if possible. This allows for that and will inform the user of the new recommended token/mixin if provided.